datapackage_pipelines_ckan/processors/dump/to_ckan.py

```diff
@@ -0,0 +1,222 @@
+import os
+import json
+import hashlib
+
+import datapackage as datapackage_lib
+from ckan_datapackage_tools import converter
+from datapackage_pipelines.lib.dump.dumper_base import FileDumper, DumperBase
+
+from datapackage_pipelines_ckan.utils import make_ckan_request
+
+import logging
+log = logging.getLogger(__name__)
+
+
+class CkanDumper(FileDumper):
```

Comment:
Do we need to inherit from FileDumper? I'm asking because it seems we're re-implementing all methods from `FileDumper`, except the `prepare_datapackage()`, which I'm not sure is useful (is it?). If we can inherit from `DumperBase`, we might be able to DRY the `handle_resources()` method (see my other comment below).

Reply:
The main thing it reuses is `handle_resource`.